classes/issue/Collector.php

```diff
@@ -441,6 +442,18 @@ public function getQueryBuilder(): Builder
                     );
                 }
             }
+
+            $pattern = '/^\d+\./i';
+            $isSearchPhraseDoi = preg_match($pattern, $this->searchPhrase);
+
+            if ($isSearchPhraseDoi) {
+                $q->orWhereIn('i.issue_id', function (Builder $query) {
+                    $query->select('is.issue_id')
+                        ->from('issues as is')
+                        ->join('dois AS d', 'is.doi_id', '=', 'd.doi_id')
+                        ->where('d.doi', 'LIKE', "{$this->searchPhrase}%");
```

Comment:
You could consider using the `whereLike` Eloquent method to keep the syntax DB agnostic (not sure it matters in this case, but thought I'd throw it out there anyway). https://laravel.com/docs/11.x/queries#additional-where-clauses